### PR TITLE
LMA-150

### PR DESCRIPTION
**Pull request by SandraPG85**
Add card menu to send a project to the trash

### CODE REVIEW

File: VAS.Core/ViewModel/MenuNodeVM.cs [hunk 1]
```diff
@@ -77,6 +78,12 @@ protected override void DisposeManagedResources ()
 			private set;
 		}
 
+		/// <summary>
+		/// Gets the color of the menu item.
+		/// </summary>
+		/// <value>The color of the menu item.</value>
+		public Color ItemColor { get; set; }
```
Comment (vguzmanp) [inline VAS.Core/ViewModel/MenuNodeVM.cs]: Shouldn't it be just `Color`?

File: VAS.Services/ViewModel/ProjectsManagerVM.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ public ProjectsManagerVM ()
 			DeleteCommand = new AsyncCommand (Delete, () => Selection.Any ());
 			SaveCommand = new AsyncCommand (Save, () => LoadedProject?.Model != null && LoadedProject.IsChanged);
 			ExportCommand = new AsyncCommand (Export, () => Selection.Count == 1);
+			TrashCommand = new AsyncCommand<TViewModel> (Trash, () => true) { IconName = "lma-trash" };
```
Comment (vguzmanp) [inline VAS.Services/ViewModel/ProjectsManagerVM.cs]: Why is this a different command than Delete?

Comment (SandraPG85) [reply VAS.Services/ViewModel/ProjectsManagerVM.cs]: the delete operation works with the selection and allows the multiple project deletion, instead this one only removes the project linked to the trash command

Comment (vguzmanp) [reply VAS.Services/ViewModel/ProjectsManagerVM.cs]: But in the future we will have multiselection for deletion too, we should just use the same command and create the selection if it's only one.

**Pull request merged**